### DISCUSSION TITLE
Approved GitHub Pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135083] [GitHub Education]
### Hi everyone,

I was approved of GitHub student developer pack and it said that the benefits will be available soon! I am just curious where will I be able to find them? Will GitHub notify me when it is ready? Also I think I missed out something how long does this GitHub student dev pack last? Thank you
![image_2024-08-08_153032658](https://github.com/user-attachments/assets/b05b7d53-f5cb-441b-a93d-874d7ded70a0)

**Comment by davevad93** [comment]
Hi @NashIvanRamos ,

Student benefits last for two years, after which you'll have to reverify to renew them.

The waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @NashIvanRamos was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by ghost** [reply]
Thank you so much @davevad93

**Reply by davevad93** [reply]
You're welcome @NashIvanRamos !

**Comment by Chaitanya029** [comment]
Hi @NashIvanRamos 

Congratulations on getting approved for the GitHub Student Developer Pack!

To answer your questions:

You can find the benefits of the GitHub Student Developer Pack in your GitHub account. Once the benefits are activated, you'll be able to access them from your account.
GitHub will not notify you explicitly when the benefits are ready. However, it usually takes 1-2 business days (up to 72 hours) after approval for the benefits to be activated.
As for the duration of the GitHub Student Developer Pack, I don't have that information. I recommend checking the GitHub documentation or reaching out to their support team for more information on that.

**Comment by UtsavSingh2902** [comment]
You can resolve a conversation in a pull request if you opened the pull request or if you have write access to the repository where the pull request was opened.

To indicate that a conversation on the Files changed tab is complete, click Resolve conversation.
